rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -106,6 +106,91 @@
         }
       }
     },
+    "type": {
+      "title": "Item type",
+      "type": "string",
+      "enum": [
+        "standard",
+        "issue"
+      ],
+      "default": "standard"
+    },
+    "issue": {
+      "title": "Issue information",
+      "type": "object",
+      "required": [
+        "display_text",
```

Comment:
Is it "displayed_text"? If so, then other occurrences have to be amended, of course.

Reply:
I used `display_text` to be consistent with `holding.patterns.issue_display_text`